### PR TITLE
update the readme to reflect the latest version

### PR DESCRIPTION
**Pull request by devoncarew**
- update the readme to reflect the latest version

### CODE REVIEW

**Comment by devoncarew** [comment]
cc @isoos for review

I don't know why this repo doesn't use a traditional changelog.md file; perhaps for visibility on the github action's marketplace page? I don't know whats typical for actions -

File: README.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
-      - uses: dart-lang/setup-dart@v1.3
+      - uses: dart-lang/setup-dart@v1.4
```
Comment (isoos) [inline README.md:57]: Maybe we can use `@v1`? /cc @mit-mit

**Review by isoos — APPROVED** [review]
lgtm

File: README.md (hunk 1 above)
Comment (jonasfj) [reply README.md]: Yeah, that's what I would suggest users too.

That way they get the latest logic...

**Review by jonasfj — APPROVED** [review]

**Pull request merged**